main.c

```diff
@@ -566,7 +566,7 @@ void run_key_handler(const char *key, unsigned int mask)
 	redraw();
 }
 
-#define MODMASK(mask) ((mask) & (ShiftMask|ControlMask|Mod1Mask))
+#define MODMASK(mask) ((mask) & (~ignore_mask))
```

Comment:
I don't think we need () around ignore mask anymore since the precedence of `~` is higher than `&`

```suggestion
#define MODMASK(mask) ((mask) & ~ignore_mask)
```